results/proto/api.proto

```diff
@@ -7,6 +7,7 @@ import "taskrun.proto";
 option go_package = "./proto";
 
 service Results {
+  rpc CreateTaskRun (CreateTaskRunRequest) returns (CreateTaskRunResponse) {}
```

Comment:
Let's follow https://google.aip.dev/133 here, and return the TaskRun as the response.

Reply:
Okay